Objects/EventItem/Chicken.gd

```diff
@@ -0,0 +1,13 @@
+extends "res://Objects/EventItem/EventItem.gd"
+
+var chicken_speed: float = -100
+
+func _ready():
+	set_start_position(1100,575)
+	move_speed = chicken_speed
+
+func _process(delta):
+	move(delta)
+
+func _on_Item_pressed():
+	._on_Item_pressed()
```

Comment:
Do we need this line? If only action after pressing item is called parent function, I think, parent function will be called automaticly if we remove line 12 and 13 from this script.

Reply:
Yes, in this script we don't need override method but in mag.gd we can't destroy mag node so _on_Item_pressed method must be override. thanks